README.md

```diff
@@ -283,6 +156,6 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ Required ] I think it would be better if you add a `LICENSE` file to the root directory of your project and also `update` the license link on the readme file so that anyone who would like to use your code knows how the project is licensed. Currently, when I click on the License link on the readme file I get a `404` page not found error, please rectify this.

_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/)_